### PR TITLE
add samconfig

### PR DESCRIPTION
**Pull request by fabiocesari**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: samconfig.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version = 0.1
```
Comment (perifaws) [inline samconfig.toml:1]: Can we remove that from the PR? That's the case the `.gitignore` prevents :)

**Review by perifaws — CHANGES_REQUESTED** [review]
Hi, just one file to remove and good to go

File: samconfig.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "aws-batch-metrics"
+s3_bucket = "aws-sam-cli-managed-default-samclisourcebucket-15sh4i0bj0yfx"
+s3_prefix = "aws-batch-metrics"
+region = "eu-west-1"
```
Comment (adamantike) [inline samconfig.toml:8]: Is there a way to not hardcode the region, and instead have SAM use the AWS account's default region?

**Comment by perifaws** [comment]
Closing due to no update.

**Pull request closed without merging**